media-gfx/blender/blender-3.2.2.ebuild

```diff
@@ -203,6 +205,11 @@ src_prepare() {
 	mv release/freedesktop/icons/symbolic/apps/blender-symbolic.svg release/freedesktop/icons/symbolic/apps/blender-${BV}-symbolic.svg || die
 	mv release/freedesktop/blender.desktop release/freedesktop/blender-${BV}.desktop || die
 
+	if use rocm; then
+		local libhip_path=$(ldconfig -p | grep "libamdhip64.so$" | awk '{print $NF}')
+		sed -e "s|/opt/rocm/hip/lib/libamdhip64.so|${libhip_path}|" -i extern/hipew/src/hipew.c || die
```

Comment:
Hmm, I'll try to talk to upstream about this as I don't think hardcoding `/opt/rocm/hip/lib/libamdhip64.so` is nice.
Of course we will have to use workarounds like this until upstream fixes it.
But I want to see if we can get an official path in.

If you feel like it perhaps you can propose a path upstream?
Otherwise I'll look into it.